sdk/monitor/azingest/README.md

```diff
@@ -1,4 +1,5 @@
 # Azure Monitor Ingestion client module for Go
+> Deprecated: use github.com/Azure/azure-sdk-for-go/sdk/monitor/ingestion/azlogs instead
```

Comment:
```suggestion
> Deprecated: use [github.com/Azure/azure-sdk-for-go/sdk/monitor/ingestion/azlogs](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/monitor/ingestion/azlogs) instead
```

Reply:
fixed in [6d6bd0c](https://github.com/Azure/azure-sdk-for-go/pull/22578/commits/6d6bd0c245f46a4da2b090ab294c1fa51e0d5e6e)